formats/specialist_document/publisher/details.json

```diff
@@ -265,6 +283,27 @@
             }
           ]
         },
+        "locations": {
```

Comment:
Do locations still need to be there? They were the precursor to postcode search (which uses GSS codes), and I thought they weren't used any more (and hoped we'd got rid of them a while back).